File: backend/src/main/java/com/ll/hotel/domain/review/review/dto/ReviewDto.java
package com.ll.hotel.domain.review.review.dto;

import com.ll.hotel.domain.review.review.entity.Review;

import java.time.LocalDateTime;

public record ReviewDto(
        Long reviewId,
        Integer rating,
        String content,
        LocalDateTime createdAt
) {
    public ReviewDto(Review review) {
        this(review.getId(), review.getRating(), review.getContent(), review.getCreatedAt());
    }
}

File: backend/src/main/java/com/ll/hotel/domain/review/review/dto/ReviewDto.kt
package com.ll.hotel.domain.review.review.dto

import com.ll.hotel.domain.review.review.entity.Review
import java.time.LocalDateTime

data class ReviewDto(
    val reviewId: Long,
    val rating: Int,
    val content: String,
    val createdAt: LocalDateTime
) {
    constructor(review: Review) : this(review.id, review.rating, review.content, review.createdAt)
}
